### PR TITLE
Add note for contributors on using hidden:true in yaml files

### PR DESCRIPTION
**Pull request by m-green**
Part of https://github.com/alphagov/govuk-frontend/issues/1830

As part of [adding docs on testing with HTML fixtures](https://github.com/alphagov/govuk-frontend-docs/pull/74), this adds a note in our contributing docs about using `hidden: true` in yaml files if you want your example to appear in the fixtures files but not the review app.

### CODE REVIEW

File: docs/contributing/testing.md
Comment (m-green) [reply docs/contributing/testing.md]: Good point! I've clarified this with a rejig in 51a6942.

File: docs/contributing/testing.md
Comment (hannalaakso) [inline docs/contributing/testing.md:40]: Would it be helpful to say that they would probably want the example to appear in the review app if there's a visual element to it?

Comment (m-green) [reply docs/contributing/testing.md:40]: Is that already covered in the first line of this section maybe? "You should add an example to the review app if the existing examples do not reflect the changes you've made."

Comment (hannalaakso) [reply docs/contributing/testing.md:40]: We should actually update that line too as not all examples appear in the review app with this new change. Maybe something along the lines of:
> You should add a new example if the existing examples do not reflect the changes you've made.

And then 

> If the example doesn't involve a visual change you can use `hidden: true` in your example to hide the example in the review app.

Comment (m-green) [reply docs/contributing/testing.md:40]: Ah I see what you mean. I'm thinking this through... is it likely that an external contributor would add an example to the yaml file that would need `hidden: true`? Or is it mainly us that would be doing that? 

I'm just wondering in retrospect whether to get into `hidden: true`, because including it might mean turning this section upside-down (because it's currently primarily focused on the review app).

Comment (hannalaakso) [reply docs/contributing/testing.md:40]: I think the rest of this section is still good in terms of the references to the review app. 

If we added the above suggestions and the contributor isn't sure what we mean by 'visual change' they could ask us.

File: docs/contributing/testing.md
Comment (hannalaakso) [inline docs/contributing/testing.md:40]: I wonder if this new paragraph would make more sense being positioned after
> 2. Add or update examples in the examples list.

as it's also about adding examples. 

The 'If you've created a new component' paragraph feels like it starts a new topic as it's about adding a new file

Comment (m-green) [reply docs/contributing/testing.md:40]: That makes senses, thanks - will tweak.

**Comment by m-green** [comment]
Rejigged this in 99267dc following a chat with @36degrees - let me know if it's all ok!

File: docs/contributing/testing.md
Comment (m-green) [reply docs/contributing/testing.md]: Thanks Hanna - I've rejigged in 99267dc following your feedback and a chat with @36degrees. Hope it all makes sense now.

**Review by vanitabarrett — APPROVED** [review]
Looks good to me 👍 Ready to merge once @hannalaakso has approved too

**Review by hannalaakso — APPROVED** [review]
Nice, a great tweak to the original content @m-green 📝

**Pull request merged**
